include/simdjson/common_defs.h

```diff
@@ -119,7 +119,15 @@ constexpr size_t DEFAULT_MAX_DEPTH = 1024;
   #define SIMDJSON_DISABLE_DEPRECATED_WARNING SIMDJSON_DISABLE_GCC_WARNING(-Wdeprecated-declarations)
   #define SIMDJSON_POP_DISABLE_WARNINGS _Pragma("GCC diagnostic pop")
 
-  #define SIMDJSON_DLLIMPORTEXPORT
+  #if defined(_MSC_VER) && defined(__clang__)
```

Comment:
This is good.

Reply:
by the way, gcc supposed dllimport/dllexport. Perhaps it should always be used. `SIMDJSON_DLLIMPORTEXPORT` should be renamed to `SIMDJSON_API`. But that's all for another change

Reply:
Yes, make it a distinct issue, please.